### ISSUE TITLE
Re-enable button recipes with zero-option dice

### ISSUE THREAD

**Issue by blackshadowshade** [buttonmen-dev/buttonmen#2772]
Now that #2641 has been accepted, it's time to re-enable the blackshadowshade Fanatic button, as well as allowing CustomBM to use zero-option dice.

**Comment by cgolubi1** [comment]
We think there might have been another problem, but we don't remember any details.  Let's throw a bunch of RandomAI tests at this, and eventually lead to a responderTest, and hope we can put this to bed.

**Comment by blackshadowshade** [comment]
PoshFrosh is currently looking at making a Fanatic and has indicated interest in using a turbo option die with zero as one of the options. #2641 should have fixed the basic issue with zero option dice, so we should now be able to address half of this issue, namely re-enabling CustomBM support for zero option.

**Comment by PoshFrosh** [comment]
> PoshFrosh is currently looking at making a Fanatic and has indicated interest in using a turbo option die with zero as one of the options. #2641 should have fixed the basic issue with zero option dice, so we should now be able to address half of this issue, namely re-enabling CustomBM support for zero option.

Yes, enabling zero-option would be nice for customBM and possibly my future fanatic. Thanks!
But I want to note for future reference that there is at least one other issue preventing @blackshadowshade 's fanatic from being re-enabled:
Turbo and konstant on the same die is disabled on staging as per #2678 and bss has mkht(T)!

**Comment by blackshadowshade** [comment]
Good pick, I had entirely forgotten about that one.
